{"nick":"Something12","date":"2016-01-13T00:32:00.632Z","type":"join"}
{"nick":"DarkGod","reason":"Quit: Leaving","date":"2016-01-13T03:04:08.111Z","type":"quit"}
{"nick":"SkyRocknRoll","date":"2016-01-13T07:45:32.007Z","type":"join"}
{"nick":"devurandom","reason":"Remote host closed the connection","date":"2016-01-13T07:47:31.952Z","type":"quit"}
{"nick":"devurandom","date":"2016-01-13T07:47:46.933Z","type":"join"}
{"nick":"DarkGod","date":"2016-01-13T08:39:43.150Z","type":"join"}
{"nick":"technomancy","date":"2016-01-13T09:56:12.247Z","type":"join"}
{"nick":"Something12","reason":"Read error: Connection reset by peer","date":"2016-01-13T10:25:54.042Z","type":"quit"}
{"nick":"technomancy","message":"https://github.com/squeek502/luvit-irc-activity-bot <- when I try to run this, it simply exits. if I add an HTTP server, it stays open, but it never prints any of the connection messages. any ideas what I'm doing wrong?","date":"2016-01-13T12:34:41.687Z","type":"message"}
{"nick":"technomancy","message":"squeek: should this work on the latest luvit?","date":"2016-01-13T12:35:12.757Z","type":"message"}
{"nick":"technomancy","message":"aha, apparently it assumes that every response from dns.resolve4 has an address, but in actuality the cname entries don't","date":"2016-01-13T12:47:40.525Z","type":"message"}
{"nick":"technomancy","message":"then there's also a bug where it calls string.join on a table with a nil first entry","date":"2016-01-13T12:50:11.447Z","type":"message"}
{"nick":"technomancy","message":"now it's only timing out","date":"2016-01-13T12:53:39.974Z","type":"message"}
{"nick":"technomancy","message":"pull request sent","date":"2016-01-13T13:25:52.705Z","type":"message"}
{"nick":"SkyRocknRoll","reason":"Ping timeout: 245 seconds","date":"2016-01-13T14:38:30.080Z","type":"quit"}
{"nick":"SkyRocknRoll","date":"2016-01-13T14:50:37.627Z","type":"join"}
{"nick":"rphillips","message":"morning","date":"2016-01-13T15:07:59.420Z","type":"message"}
